src/ergw_aaa.erl

```diff
@@ -112,7 +112,7 @@ get_service(Name) ->
 
 get_application(Name) ->
     Apps = application:get_env(ergw_aaa, ?APP_KEY, #{}),
-    maps:get(Name, Apps, #{}).
+    maps:get(Name, Apps, #{procedures => #{}}).
```

Comment:
so, procedure is not mandatory, then the validation in https://github.com/travelping/ergw_aaa/blob/master/src/ergw_aaa_config.erl#L125 should check that too.

Adding a default should be enough. e.g. change L125 in ergw_config to

```
maps:map(validate_app_option(App, _, _ ), maps:merge(#{procedures => #{}}, AppOptions));
```

Reply:
ok, thanks for the suggestion